app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,224 @@
+<resources>
+    <string name="app_name">GreenStash</string>
+
+    <!-- Other Strings -->
+    <string name="confirm">Conferma</string>
+    <string name="cancel">Cancella</string>
+    <string name="ok">Ok</string>
+    <string name="unknown_error">Ops! Qualcosa è andato storto.</string>
+
+    <!-- Navigation Drawer -->
+    <string name="drawer_home">Inizio</string>
+    <string name="drawer_archive">Archivio</string>
+    <string name="drawer_backup">Backup</string>
+    <string name="drawer_settings">Impostazioni</string>
+    <string name="drawer_rating">Valutaci</string>
+    <string name="drawer_share">Condividi</string>
+    <string name="drawer_privacy">Privacy</string>
+    <string name="drawer_footer_text">Fatto con ❤️ in India.</string>
+    <string name="drawer_share_message">
+        Hey! Dai un'occhiata a quest'app fantastica chiamata GreenStash.
```

Comment:
Escape (`'`) chars here, i.e. `un'occhiata` -> `un\'occhiata` same for the other one.